source/util/AMErrorMonitor.h

```diff
@@ -161,9 +163,11 @@ class AMErrorMon : public QObject {
 	}
 
 	/// Report an alert level AMErrorReport.  Builds an AMErrorReport using the information provided. This function is thread-safe.
-	static void alert(const QObject *src = 0, int code = 0, const QString &desc = "")
+	static void alert(const QObject *src = 0, int code = 0, const QString &desc = "", bool showWarningMessage=false)
```

Comment:
I can see the value of this, but if it's going to be available for alert, then it should be available for the other ones as well.